can/interfaces/tosun.py

```diff
@@ -0,0 +1,274 @@
+import collections
+import time
+import warnings
+from typing import List, Optional, Tuple, Union, Deque, Any
+
+import can
+import can.typechecking
+import ctypes
+from can.bus import LOG
+from can.exceptions import (
+    CanError,
+    CanInterfaceNotImplementedError,
+    CanOperationError,
+    CanInitializationError,
+)
+
+from tosun import TSCanMessage, TSCanFdMessage, TSMasterException, TosunDevice, TSMasterMessageType
```

Comment:
> from tosun import TSCanMessage, TSCanFdMessage, TSMasterException, TosunDevice, TSMasterMessageType 
How can I find/install `tosun` package?

Reply:
The tosun package is planing to be refactored by using rust